src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -94,18 +95,23 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
+
     result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
 
+    if no_wait:
+        return result_poller
+
     return _handle_result(cmd, result_poller, source_registry, source_image, registry)
 
 
 def _handle_result(cmd, result_poller, source_registry, source_image, registry):
     from msrestazure.azure_exceptions import ClientException
     try:
-        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...')(result_poller)
+        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...', 'Import has finished')(result_poller)
+
     except CLIError as e:
         try:
             # if command fails, it might be because user specified registry twice in --source and --registry
```

Comment:
finally let's make sure that with or without no-wait (if user enters input incorrectly) and service rejects the request in the event of no-wait, we show the user the warning below..